docs/RFCs/package-toml.md

```diff
@@ -0,0 +1,166 @@
+# Manifest TOML format
+
+A proposal for a more formal TOML definition for entropic
+
+- Version: DRAFT
+- Date: 06/06/2019
+- Authors
+  - Todd Kennedy <todd@selfassembled.org>
+
+## Abstract
+
+This document is meant to describe the package manifest format for Entropic,
+a federated solution to package management. Entropic needs to have its own
+manifest file due to conflicts arising from the namespacing entropic uses.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Copyright
+
+This document is copyright by its individual contributors, and available for
+re-use under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+## Introduction
+
+Entropic cannot share a manifest format with the traditional `package.json` format
+associated with the ECMAScript ecosystem for a number of reasons:
+
+1. Entropic is intended to allow for federated package management for more than just the ECMAScript ecosystem
+1. The `package.json` name specifier does not allow for the namespacing that entropic provides
+1. There are recognized deficiencies with using JSON as human-readable metadata format
+
+The project has chosen to standardize on TOML as a format due to various short-comings in the JSON file specification, especially concerns around:
+
+1. Lack of comments
+1. Lack of multiline string support
+1. Designed for ease of generation by machine, not by hand
+
+## Filename
+
+The filename of the manifest **MUST** be `[TK].toml` and it **MUST** live in
```

Comment:
Can a package have multiple roots? That is currently the case for projects in the Node.js ecosystem where it is possible to have nested projects

Reply:
Yes, there will be support for monorepo (ncluding the location of this package within the monorepo).  

This should work like it does with the existing package manager.